Sources/StreamVideoSwiftUI/CallView/ParticipantsGridView.swift

```diff
@@ -8,37 +8,40 @@ import WebRTC
 
 @MainActor
 struct ParticipantsGridView<Factory: ViewFactory>: View {
-    
+
     var viewFactory: Factory
     var call: Call?
     var participants: [CallParticipant]
     var availableSize: CGSize
     var isPortrait: Bool
     var participantVisibilityChanged: (CallParticipant, Bool) -> Void
-    
+
     var body: some View {
-        ScrollView {
-            if #available(iOS 14.0, *) {
-                LazyVGrid(
-                    columns: [
-                        .init(.adaptive(minimum: size.width, maximum: size.width), spacing: 0)
-                    ],
-                    spacing: 0
-                ) {
-                    participantsContent
-                }
-                .frame(width: availableSize.width)
-            } else {
-                VStack {
-                    participantsContent
+        GeometryReader { geometryProxy in
+            ScrollView {
+                if #available(iOS 14.0, *) {
+                    LazyVGrid(
+                        columns: [
+                            .init(.adaptive(minimum: size.width, maximum: size.width), spacing: 0)
+                        ],
+                        spacing: 0
+                    ) {
+                        participantsContent(geometryProxy.frame(in: .global))
```

Comment:
This approach also makes the usage a bit tied to our grid component. What about the spotlight and other views that customers might want to use in their apps?

Reply:
Spotlight also has this updated. What part of the approach seems to you that is tied to the Grid?

Reply:
Where exactly is this modifier used in spotlight? I don't see it.